scripts/getDependencies.pl

```diff
@@ -129,6 +129,20 @@
 		fname => 'org.eclipse.osgi-3.16.100.jar',
 		sha1 => '7ddb312f386b799d6e004d193a01c50169bf69f3'
 	},
+	jtreg_4_2_0_tip => {
```

Comment:
Thanks @vanichitkara 
I would suggest having the `jtreg_4_2_0_tip` and `jtreg-4.2.0-tip.tar.gz` be the same. Otherwise, we will need to reference two different names in adoptium/aqa-tests#2545 
That is, set fname to be `jtreg_4_2_0_tip.tar.gz`

Reply:
Changed the fname of both the versions accordingly.

Reply:
Thanks @vanichitkara